java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -4148,6 +4148,93 @@ void testMergeApproxPercentile2() {
     }
   }
 
+  @Test
+  void testCreateTDigestReduction() {
+    try (Table t1 = new Table.TestBuilder()
+            .column(100, 150, 160, 70, 110, 160)
+            .build();
+         Scalar tdigest = t1.getColumn(0)
+            .reduce(ReductionAggregation.createTDigest(1000), DType.STRUCT)) {
+      assertEquals(DType.STRUCT, tdigest.getType());
+      ColumnView[] columns = tdigest.getChildrenFromStructScalar();
+      assertEquals(3, columns.length);
+      try (HostColumnVector centroids = columns[0].copyToHost();
+          HostColumnVector min = columns[1].copyToHost();
+          HostColumnVector max = columns[2].copyToHost()) {
+        assertEquals(DType.LIST, centroids.getType());
+        assertEquals(DType.FLOAT64, min.getType());
+        assertEquals(DType.FLOAT64, max.getType());
+        assertEquals(1, min.getRowCount());
+        assertEquals(1, max.getRowCount());
+        assertEquals(70, min.getDouble(0));
+        assertEquals(160, max.getDouble(0));
+      }
+    }
+  }
+
+  @Test
+  void testMergeTDigestReduction() {
+    StructType centroidStruct = new StructType(false,
+            new BasicType(false, DType.FLOAT64), // mean
+            new BasicType(false, DType.FLOAT64)); // weight
+
+    ListType centroidList = new ListType(false, centroidStruct);
+
+    StructType tdigestType = new StructType(false,
+            centroidList,
+            new BasicType(false, DType.FLOAT64), // min
+            new BasicType(false, DType.FLOAT64)); // max
+
+    try (ColumnVector tdigests = ColumnVector.fromStructs(tdigestType,
+            new StructData(Arrays.asList(
+                    new StructData(1.0, 100.0),
+                    new StructData(2.0, 50.0)),
+                    1.0, // min
+                    2.0), // max
+            new StructData(Arrays.asList(
+                    new StructData(3.0, 200.0),
+                    new StructData(4.0, 99.0)),
+                    3.0, // min
+                    4.0)); // max
+        Scalar merged = tdigests.reduce(ReductionAggregation.mergeTDigest(1000), DType.STRUCT)) {
+
+      assertEquals(DType.STRUCT, merged.getType());
+      ColumnView[] columns = merged.getChildrenFromStructScalar();
```

Comment:
The views in `columns` need to be closed or a native host memory leak occurs. `ClosableArray` might be useful here, e.g.:
```java
  try (CloseableArray views = CloseableArray.wrap(merged.getChildrenFromStructScalar()) {
``